packages/teams-js/.eslintrc.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
   ignorePatterns: ['.eslintrc.js'],
   parserOptions: {
-    project: './tsconfig.strictNullChecks.json',
+    project: './tsconfig.eslint.json',
```

Comment:
Eslint will now share the same tsconfig as the used when compiling (with one minor change that you will see in the eslint tsconfig file)